linux_os/guide/system/software/integrity/software-integrity/aide/file_audit_tools_permissions/tests/missing_file_test.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I think you can move this test scenario to `shared/templates/file_permissions/tests` and make it remove all paths listed in `filepath`.